elements/pfe-sass/functions/_custom-properties.scss

```diff
@@ -141,17 +104,17 @@
 /// @param {String} $broadcast - name of the broadcast variable to apply
 /// @requires {String} $repo - Name of repo, which is "pfe"
 ///
-/// @example:   color: pfe-broadcasted(ui-base--hover);
+/// @example:   color: #{pfe-broadcasted(ui-base--hover)};
```

Comment:
Do not add back the interpolation - it is not needed.  Can also remove whitespace that was added below.

Reply:
I think this was a merge issue, I don't remember messing with comments on existing functions. Reverted this.